docs/src/pages/guides/FHIRServerUsersGuide.md

```diff
@@ -2594,7 +2594,7 @@ For more information about topics related to configuring a FHIR server, see the
 
 - <b id="f5">5</b>
 
-    An external reference is a reference to a resource which is meaningful outside a particular request bundle.  The value typically includes the resource type and the resource identifier, and could  be an absolute or relative URL. Examples:  `https://fhirserver1:9443/fhir-server/api/v4/Patient/12345`, `Patient/12345`, etc. [↩](#a5)
+    An external reference is a reference to a resource which is meaningful outside a particular request bundle.  The value must include the resource type and the resource identifier, and could be an absolute or relative URL. Examples: `https://fhirserver1:9443/fhir-server/api/v4/Patient/12345`, `Patient/12345`, etc. [↩](#a5)
```

Comment:
We still support absolute URLs of any pattern, right?  Like we don't validate that the URL pattern happens to follow the FHIR pattern, right?

Reply:
Previous behavior:
- If anything (rsrcType, etc.) in the reference was invalid, it would skip validation of the pattern.
Current behavior:
- If anything (rsrcType, etc.) in the reference is invalid (and it doesn't start with '#' or 'urn:'), it will detect it as invalid.

Sounds like there's a different behavior in mind. Such as if it starts with "http:" or "https:", to skip validation?

Reply:
> Sounds like there's a different behavior in mind. Such as if it also starts with "http:" or "https:", to skip validation?

yes, that would get my vote

Reply:
ok, I'll do that.  That should also allow me to revert all the spec example changes, since those references started with "http:" or "https:".

Reply:
Updated.